x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.tsx

```diff
@@ -47,6 +48,45 @@ import { MANAGEMENT_APP_ID } from '../../../common/constants';
 import { PolicyDetailsRouteState } from '../../../../../common/endpoint/types';
 import { WrapperPage } from '../../../../common/components/wrapper_page';
 import { HeaderPage } from '../../../../common/components/header_page';
+import { PolicyAdvanced } from './policy_advanced';
+import * as AdvancedPolicySchema from '../models/advanced_policy_schema.json';
+
+interface AdvancedPolicySchemaType {
+  key: string;
+  first_supported_version: string;
+  last_supported_version: string;
+}
+
+const AdvancedPolicyForms = React.memo(() => {
+  return (
+    <EuiFlexGrid columns={3}>
+      <EuiFlexItem>
+        <h1>{'Field name'}</h1>
```

Comment:
I'm curious why this has to be written as `<h1>{'Field name'}</h1>` instead of `<h1>Field name</h1>`. The latter was producing the error: `Missing JSX expression container around literal string`

Reply:
For this, you should use the `FormattedMessage` for internationalization since this is a static string.  It would look something like this.

```js
<FormattedMessage
   id="xpack.securitySolution.policyAdvanced.fieldName"
   defaultMessage="Field Name"
/>
```

Reply:
oh, i see. `FormattedMessage`s are for internationalization.

Reply:
23238e9d29504f34a9e774cad99443b830951c52